### PR TITLE
added support magma version tags

### PR DESCRIPTION
**Pull request by Two-Faces**
There is a problem with constant updates, which can theoretically break the server startup. I checked the launch of the desired version and it works fine.

### CODE REVIEW

**Comment by Two-Faces** [comment]
The parameter, in theory, is not required and can be left empty in order to always take the latest version.

File: README.md [hunk 1]
```diff
@@ -511,8 +511,10 @@ Extra variables:
 A [Magma](https://magmafoundation.org/) server, which is a combination of Forge and PaperMC, can be used with
 
     -e TYPE=MAGMA
+    -e MAGMA_VERSION=9f3a3c25
 
-> **NOTE** there are limited base versions supported, so you will also need to  set `VERSION`, such as "1.12.2", "1.16.5", etc.
+> **NOTE** there are limited base versions supported, so you will also need to  set `VERSION`, such as "1.12.2", "1.16.5", etc. 
+> To switch the MAGMA version, you need to take the name of the desired tag.
```
Comment (itzg) [inline README.md:517]: Can you rephrase "you need to take the name of the desired tag" and/or explain how one obtains that? As written I have no idea what you're talking about, but looking at the example I'm having to guess you mean the git revision of Magma.

File: scripts/start-deployMagma [hunk 2]
```diff
@@ -6,7 +6,7 @@ isDebugging && set -x
 
 resolveVersion
 
-if ! downloadUrl=$(get --json-path '$.link' "https://api.magmafoundation.org/api/v2/${VERSION}/latest"); then
+if ! downloadUrl=$(get --json-path '$.link' "https://api.magmafoundation.org/api/v2/${VERSION}/latest/${MAGMA_VERSION}"); then
```
Comment (itzg) [inline scripts/start-deployMagma:9]: I know this script doesn't follow the pattern yet, but please do like most of the other scripts and pre-declare this variable and its default of being empty, such as

https://github.com/itzg/docker-minecraft-server/blob/a6c3943136f6c32e5a1e69488379917e330656c2/scripts/start-deployAutoCF#L7C6-L7C6

**Review by itzg — COMMENTED** [review]
Thanks! Just a couple of small items.

**Review by itzg — APPROVED** [review]
Thanks for the quick updates
